data/binding/queue.go

```diff
@@ -1,6 +1,6 @@
 package binding
 
-var itemQueue = make(chan itemData, 1024)
+var itemQueue = make(chan itemData, 10000024)
```

Comment:
Is this change related to form hint and error text?

Reply:
Good catch sorry.